account_multicurrency_revaluation_rate_type/wizard/wizard_currency_revaluation.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class WizardCurrencyRevaluationType(models.TransientModel):
+
+    _inherit = 'wizard.currency.revaluation'
+
+    @api.model
+    def _get_default_rate_type(self):
+        """
+        Get default rate type if one is defined in company settings
+        """
+        return self.env.user.company_id.revaluation_rate_type
+
+    revaluation_rate_type = fields.Selection(
+        string='Rate type',
+        selection=[
+            ('average', 'Monthly Average'),
+            ('daily', 'Daily'),
+        ],
+        default=lambda self: self._get_default_rate_type(),
+    )
+
+    @api.model
+    def _compute_unrealized_currency_gl(self, currency_id, balances, form):
+        if self.revaluation_rate_type == 'average':
+            self = form.with_context(monthly_rate=True)
```

Comment:
Seeing this made me realize we don't really need `form` parameter. 
I added another commit here : https://github.com/OCA/account-closing/pull/102/commits/82056c3e5cee0c42a3aa729bf25d8065e5e8e6ad

Reply:
ok,  so lets merge https://github.com/OCA/account-closing/pull/103 first as it's obviously blocking